### PR TITLE
Updated balance component on homeview

### LINKED ISSUE #136: Improve general UX
Idea is to remove the balance page (for now, we can statistics later), but we can view the balance and do top ups direct on the mainpage.

### PR DESCRIPTION
**Pull request by SuperVK**
Updated balance component for better UX, such as instant balance increase on homeview. New balance previews, cleaner look. This removes the balance view as well as it is not needed anymore, this can be brought back if needed in the future.

### CODE REVIEW

File: src/layout/DashboardLayout.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="h-screen flex flex-column">
+  <div class="h-screen flex flex-column align-items-center">
```
Comment (CodeNamedRobin) [inline src/layout/DashboardLayout.vue:2]: Somehow, there is no margin between the bottom copyright banner and the content.

File: src/styles/BasePage.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
   font-weight: 500;
   font-family: Raleway, sans-serif !important;
   font-size: 2.5rem;
-  margin-bottom: 1.5rem;
+  margin-bottom: 1rem;
```
Comment (CodeNamedRobin) [inline src/styles/BasePage.scss]: I suspect this is causing margin issues.

**Review by CodeNamedRobin — CHANGES_REQUESTED** [review]
Some minor issues, also needs rebasing.

File: src/styles/BasePage.scss (hunk 2 above)
Comment (SuperVK) [reply src/styles/BasePage.scss]: This is only affecting the page-title so sadly  not

File: src/layout/DashboardLayout.vue (hunk 1 above)
Comment (SuperVK) [reply src/layout/DashboardLayout.vue:2]: Has been fixed!

**Comment by SuperVK** [comment]
@CodeNamedRobin can this be merged? :)

**Review by CodeNamedRobin — APPROVED** [review]
LGTM!

**Pull request merged**